src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana-details-panel/grafana-details-panel.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, EventEmitter, Input, OnChanges, OnInit, Output } from '@angular/core';
+
+import * as _ from 'lodash';
+
+import { DomSanitizer } from '@angular/platform-browser';
+import { SafeUrl } from '@angular/platform-browser';
+
+import { DashboardVar } from '../../../enum/grafana-template.enum';
+import { GrafanaDashboardName } from '../../../enum/grafana-template.enum';
+import { GrafanaPanelsRow } from '../../../enum/grafana-template.enum';
+import { GrafanaPanelsUid } from '../../../enum/grafana-template.enum';
+import { GrafanaUrl } from '../../../enum/grafana-template.enum';
+import { GrafanaVar } from '../../../enum/grafana-template.enum';
```

Comment:
Why not importing all these variables in a single line?
```ts
import { DashboardVar, GrafanaDashboardName, GrafanaPanelsRow, GrafanaPanelsUid, GrafanaUrl, GrafanaVar} from '../../../enum/grafana-template.enum';
```
Or given you have imported everything, why not  `import *`?

Reply:
Ack